File: Fortran/BuildingGeometry.f
      FaceNormalNo=5
      allocate(FaceNormals(FaceNormalNo,3))
      FaceNormals(1,1:3)=(/-1,0,0/)
      FaceNormals(2,1:3)=(/0,1,0/)
      FaceNormals(3,1:3)=(/1,0,0/)
      FaceNormals(4,1:3)=(/0,-1,0/)
      FaceNormals(5,1:3)=(/0,0,1/)

      Boxnumber=1
      allocate(Boxarraynear(boxnumber,3))
      allocate(Boxarrayfar(boxnumber,3))
      Boxarraynear(1,1:3)=(/10,10,0/)
      boxarrayfar(1,1:3)=(/64.4322,46.9316,8.2423/)
      TriangleNumber=0
      SquareNumber=0
      PolyBuilding=0

File: BuildingGeometry.py
import numpy as np

FaceNormalNo = 5
FaceNormals = [(-1, 0, 0), (0, 1, 0), (1, 0, 0), (0, -1, 0), (0, 0, 1)]
# ^Will's Code

BoxNumber = 1
BoxArrayNear = np.zeros([BoxNumber, 3])
BoxArrayFar = np.zeros([BoxNumber, 3])
BoxArrayNear[0] = [10, 10, 0]
BoxArrayFar[0] = [64.4322, 46.9316, 8.2423]
TriangleNumber = 0
SquareNumber = 0
PolyBuilding = 0
